aptos-move/block-executor/src/executor.rs

```diff
@@ -1114,17 +1134,25 @@ where
                     // Record the status indicating abort.
                     return Err(Error::UserError(err));
                 },
+                ExecutionStatus::DirectWriteSetTransactionNotCapableError => {
+                    return Err(Error::FallbackToSequential(PanicOr::Or(
+                        IntentionalFallbackToSequential::DirectWriteSetTransaction,
```

Comment:
We are already in sequential execution. What would happen differently when we fallback to sequential execution again here?

Reply:
we now have two sequential fallbacks - with dynamic_change_set_optimizations_enabled true and false.